augur/data/schema-multiplot.json

```diff
@@ -0,0 +1,126 @@
+                    "type": "array",
+                    "minItems": 1,
+                    "items": {
+                        "description": "A single collection option",
+                        "type": "object",
+                        "additionalProperties": false,
+                        "required": ["key"],
+                        "properties": {
+                            "key": {
+                                "description": "The property name of the collection",
+                                "type": "string"
+                            },
+                            "title": {
+                                "description": "The title to display in the collections dropdown. Optional -- if not provided, then `key` will be used",
+                                "type": "string"
+                            }
+                        }
+                    }
+                },
+                "groupings": {
+                    "description": "The available group by options and their display titles",
+                    "type": "array",
+                    "minItems": 1,
+                    "items": {
+                        "description": "A single group by option",
+                        "type": "object",
+                        "additionalProperties": false,
+                        "required": ["key"],
+                        "properties": {
+                            "key": {
+                                "description": "The property name of the group by option",
+                                "type": "string"
+                            },
+                            "title": {
+                                "description": "The title to display in the group by dropdown. Optional -- if not provided, then `key` will be used",
+                                "type": "string"
+                            }
+                        }
+                    }
+                },
+                "display_defaults": {
+                    "description": "Default display options of the multiplot panel",
+                    "type": "object",
+                    "additionalProperties": false,
+                    "minProperty": 1,
+                    "properties": {
+                        "group_by": {
+                            "description": "Default group by option where the value must be the `key` in one of the objects of the groupings array. Optional -- if not provided, first group by option will be used",
+                            "type": "string"
+                        },
+                        "collection": {
+                            "descriptions": "Default collection option where the value must be the `key` in one of the objects of the collections array. Optional -- if not provided, first collection option will be used",
+                            "type": "string"
+                        },
+                        "threshold": {
```

Comment:
Any thresholds will be per-collection, so I suggest placing them in the collections config like so:

```json
"collections": [
    {
        "key": "HI_raw",
        "title": "Raw HI assay measurements",
        "threshold": 2
    }
```

I don't know if this should be toggle-able (sic), John & Trevor will know more here.

> if not provided then threshold will not be included and threshold controls will be hidden

My thinking about this section is that display defaults shouldn't restrict what state the user can display.